### ISSUE TITLE
version 4.1.3, when push to a page with super-tabs, there has a white blank when push or back.

### ISSUE THREAD

**Issue by longdw** [zyra/ionic2-super-tabs#191]
I download the ionic2-super-tabs-example code, and modify the `app.module.ts`, 
```
IonicModule.forRoot(MyApp, {
      mode: 'ios',//md
      backButtonText: ''
    }),
```
modify the `page1.ts`
```
pushPage(localNavCtrl: boolean = false) {
    if (localNavCtrl) {
      this.navCtrl.push('HomePage');
    } else {
      this.rootNavCtrl.push('HomePage');
    }
  }
```

when push from HomePage to Page1Page or back from Page1Page, there has a white blank between two pages and  shows for a moment, like follow gif shows:

![result](https://user-images.githubusercontent.com/4987178/31757039-c3ccb654-b46c-11e7-94ed-1fbecc08796b.gif)

**Comment by lukawal-miller** [comment]
It works fine on Android, I've noticed the same issue on iOS devices.

**Comment by crissi** [comment]
same here

**Comment by icharge** [comment]
me too.

**Comment by ihadeed** [comment]
Unable to reproduce in iPhone X simulator. Is this still an issue with latest version of everything?

**Comment by sourbell** [comment]
I think this issue has been fixed on this commit: 
[fix(ios): slide animation bug (#235) ](https://github.com/zyra/ionic2-super-tabs/commit/a55167d21436f30033315b0ae4814cf71a9b902c)
